src/matcher.ts

```diff
@@ -161,20 +170,25 @@ export default class Matcher {
 					// source += 'if (el.id != ' + JSON.stringify(tagName.substr(1)) + ') return false;';// 1
 					function_name += '1';
 				} else {
-					reg = /^\[\s*(\S+)\s*(=|!=)\s*((((["'])([^\6]*)\6))|(\S*?))\]\s*/.exec(tagName);
+					reg = /^\[\s*(\S+)\s*(=|!=)\s*((((["'])([^\6]*)\6))|(\S*?))\]\s*/.exec(tagName)
 					if (reg) {
-						attr_key = reg[1];
-						let method = reg[2];
+						attr_key = reg[1]
+						let method = reg[2]
 						if (method !== '=' && method !== '!=') {
-							throw new Error('Selector not supported, Expect [key${op}value].op must be =,!=');
+							throw new Error('Selector not supported, Expect [key${op}value].op must be =,!=')
 						}
 						if (method === '=') {
-							method = '==';
+							method = '=='
 						}
-						value = reg[7] || reg[8];
+						value = reg[7] || reg[8]
 
 						// source += `let attrs = el.attributes;for (let key in attrs){const val = attrs[key]; if (key == "${attr_key}" && val == "${value}"){return true;}} return false;`;// 2
-						function_name += '2';
+						function_name += '2'
```

Comment:
Why have you removed `;`?
I suggest keep them.

Reply:
Oh, these are just my local prettier settings. I'll fix it